tests/application/access-policy/00-assert.yaml

```diff
@@ -59,7 +59,7 @@ spec:
 apiVersion: networking.istio.io/v1beta1
 kind: ServiceEntry
 metadata:
-  name: access-policy-egress-3a90cb5d70dc06a
+  name: access-policy-egress-3a90cb5d70dc06a-99561aeccea8b051
```

Comment:
Will ServiceEntries change name? Could that somehow cause issues with orphaned serviceentries from the last version not being deleted by skiperator?

Reply:
Yeah, that's definitely something to keep in mind. We should go over all the old service entries and delete those.

This is done due to the fact that jobs and applications would have the same naming scheme for access policies. One solution could be to append `job` or something if the type is job.

I'm not sure if you agree with how its done here, simply appending a hash of the name of the owned object.

Reply:
ServiceEntries used to be prefixed with the application name (`skjema-bruksnavn-frontend-egress-5d0c98885286d5e1` for example), which I think would solve that issue? What's the reason we moved away from that? I feel a hash isn't as clear as user friendly as a plain text name

Reply:
If you add the kind to that name I mean. Or what do you think, do you prefer a hash?

Reply:
To prevent changing existing names you could even consider the application resources a "default", only adding a `job` prefix for jobs.

Reply:
They are still prefixed with the application name. However, if a Job and an Application had the same name, there would be a naming collision for the service entry.

I think doing as you say with only pre/suffixing jobs is a solution, it just also feels sort of "unclean".. But it might be a better solution than having to clean up after *a lot* of old serviceentries